### PR TITLE
Add instuctions to install all the needed packages to compile the sou…

### PR DESCRIPTION
**Pull request by x86girl**
Add to README.md all the needed packages to successful create the build:

texlive-scheme-basic ghostscript-tools-dvipdf 'tex(tabls.sty)' 'tex(multirow.sty)' 'tex(fancybox.sty)'  \
'tex(minifp.sty)'  'tex(keystroke.sty)' 'tex(leystroke.sty)  'tex(ulem.sty)' 'tex(hypernat.sty)'  \
'tex(lettrine.sty)

### CODE REVIEW

**Comment by alxpl** [comment]
Hi Priscila,

This manual is not strictly distro-specific, so it would be good to add a header or something mentioning that these instructions pertain to Fedora.

**Comment by x86girl** [comment]
Thanks for your feedback
I added a comment to avoid misunderstanding, marking that dnf install -y ....
as Fedora only
Can you please review it again?

File: README.md [hunk 1]
```diff
@@ -5,5 +5,11 @@ This repository contains the files to build the PDF of my manual
 ## Build instructions
 
 ```
+#Fedora only
```
Comment (alxpl) [inline README.md:8]:
```suggestion
Install the necessary packages:

#Fedora:
```

File: README.md [hunk 2]
```diff
@@ -5,5 +5,11 @@ This repository contains the files to build the PDF of my manual
...
+sudo dnf -y install texlive-scheme-basic ghostscript-tools-dvipdf 'tex(tabls.sty)' 'tex(multirow.sty)' 'tex(fancybox.sty)'  \
+'tex(minifp.sty)'  'tex(keystroke.sty)' 'tex(leystroke.sty)'  'tex(ulem.sty)' 'tex(hypernat.sty)'  \
+'tex(lettrine.sty)'  
+
+#General build
```
Comment (alxpl) [inline README.md:13]:
```suggestion
#Build the pdf:
```

File: README.md [hunk 3]
```diff
@@ -5,5 +5,13 @@ This repository contains the files to build the PDF of my manual
 ## Build instructions
 
 ```
+Install the necessary packages:
+
+#Fedora:
```
Comment (alxpl) [inline README.md:10]:
```suggestion
#Fedora:
```
```

**Review by alxpl — APPROVED** [review]
I think I might have messed it up, I'll merge and untangle what's needed.

**Comment by Slookeur** [comment]
Thank you for your input, I added the Debian part ;-)

**Comment by x86girl** [comment]
;)

**Pull request merged**